### PR TITLE
refactor: update oep sponsoring

### PR DESCRIPTION
**Pull request by tiagolobocastro**
As discussed in the maintainers meeting, removing the discussions and auto issue generation.
I've tried to add some defined process, but without too many details (more on the OEP process itself which lives in openebs/openebs anyway).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -135,17 +135,15 @@ A person looking to contribute can take up an issue by claiming it as a comment/
...
-7. Approved OEPS's are moved (or copied) into the appropriate repository and are tracked against the OEP Community Roadmap
-8. A pull request (PR) is only created after this approval. Once the PR is created, the existence of the PR means engineering work is approved and the OEP is scheduled for implementation.
-
+New enhancements are referred to as OpenEBS Enhancement Proposals (OEPs).
+Anyone can sponsor a new OEP. Here is the process:
+
+1. Sponsor creates a GitHub issue to track the OEP as a whole.
+1. Sponsor creates a GitHub (pull request) PR with a `provisional` OEP, with its oep-number set to the tracking GitHub issue number.
+1. Discussions are encouraged, either on the OEP related issues or the PR.
+1. OEPs are reviewed in the maintainer and community meetings.
```
Comment (niladrih) [inline CONTRIBUTING.md:144]: Should this be the community meetings exclusively, so it's always accessible to contributors?

**Review by Abhinandan-Purkait — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -135,17 +135,15 @@ A person looking to contribute can take up an issue by claiming it as a comment/
...
-5. The sponsor presents the OEP
-6. The community votes on the OEP, maintainers have binding votes.
-7. Approved OEPS's are moved (or copied) into the appropriate repository and are tracked against the OEP Community Roadmap
-8. A pull request (PR) is only created after this approval. Once the PR is created, the existence of the PR means engineering work is approved and the OEP is scheduled for implementation.
-
+New enhancements are referred to as OpenEBS Enhancement Proposals (OEPs).
+Anyone can sponsor a new OEP. Here is the process:
+
+1. Sponsor creates a GitHub issue to track the OEP as a whole.
+1. Sponsor creates a GitHub (pull request) PR with a `provisional` OEP, with its oep-number set to the tracking GitHub issue number.
```
Comment (avishnu) [inline CONTRIBUTING.md:142]: Should the index bullet be incremental? Currently, I see all '1'?

Comment (tiagolobocastro) [reply CONTRIBUTING.md:142]: If you look at the file render you'll see it's correct

Comment (avishnu) [reply CONTRIBUTING.md:142]: It renders correctly. Strange?

**Review by avishnu — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (tiagolobocastro) [reply CONTRIBUTING.md:142]: AIUI it's expected, so you don't need to fix numbers if you add something in the middle

File: CONTRIBUTING.md (hunk 1 above)
Comment (tiagolobocastro) [reply CONTRIBUTING.md:144]: This review is just to go though the current list of OEPs and check their progress. I can see this being done more in the maintainer than the community meeting, unless someone from the community wants to bring up the issue for discussion.
Maybe "review" here is not the right term?

**Review by niladrih — APPROVED** [review]

**Pull request merged**